src/react/components/ui/sonner.tsx

```
@@ -0,0 +1,38 @@
+import {
+  CircleCheckIcon,
+  InfoIcon,
+  Loader2Icon,
+  OctagonXIcon,
+  TriangleAlertIcon,
+} from "lucide-react"
+import { useTheme } from "next-themes"
+import { Toaster as Sonner, type ToasterProps } from "sonner"
```

ESLint

[error] Expected 2 empty lines after import statement not followed by another import.

(import/newline-after-import)